### PR TITLE
Root login rule

### PR DESCRIPTION
**Pull request by Rub21**
We've created root user login rule, could you check it?

cc. @zmully @ianshward

### CODE REVIEW

File: test/rules/rootLogin.test.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  }
+};
+
+var rub21LoginEvent = {
+  "detail": {
+    "eventVersion": "1.02",
+    "userIdentity": {
+      "type": "IAMUser",
+      "principalId": "AIDAEZ7VBM6PDZEXAMPLE",
+      "arn": "arn:aws:iam::12345679012:user/root",
```
Comment (ianshward) [inline test/rules/rootLogin.test.js]: @Rub21 Where it says `"arn": "arn:aws:iam::12345679012:user/root"` it should say `"arn": "arn:aws:iam::12345679012:user/rub21"`

**Comment by ianshward** [comment]
Nice work @Rub21.  This looks good to merge once you update that fixture.

**Comment by ianshward** [comment]
@Rub21 please squash your commits before merging.

**Comment by Rub21** [comment]
@ianshward , I think it's ready to go. but not sure how many commit will merge in master!, how can i know that?

**Comment by zmully** [comment]
@Rub21 if you look at the "commits" tab for this PR, it will show you all the commits that will be appended to the master branch. Many of these should be collapsed or `squashed` into a single commit, since they do not provide any value when appended to the master branch. In git, this is called a `rebase`. Let me know if you walk to walkthrough how to clean up the commit history on this PR if the `rebase` documentation isn't helpful.

**Comment by Rub21** [comment]
it is ready to merge!

**Pull request merged**